observability/logging/logging-6.0/log6x-about.adoc

```diff
@@ -34,7 +34,23 @@ The Cluster Logging Operator manages the deployment and configuration of the col
 == Validation
 Logging includes extensive validation rules and default values to ensure a smooth and error-free configuration experience. The `ClusterLogForwarder` resource enforces validation checks on required fields, dependencies between fields, and the format of input values. Default values are provided for certain fields, reducing the need for explicit configuration in common scenarios.
 
-=== Quick Start
+== Quick Start
+
+OpenShift Logging supports two data models:
+
+* ViaQ (General Availability)
+* OTEL (Technology Preview)
+
+You can select either of these data models based on your requirements. The ViaQ data model is the default and fully supported option, while OTEL uses the OpenTelemetry standards and is not fully supported. For more information, see xref:../../../observability/logging/logging-6.0/log6x-otel-data-model.adoc#log6x-otel-data-model[OTEL data model].
+
+=== Quick Start with ViaQ
+
+To use the default ViaQ data model, follow these steps:  
+
+[NOTE]
+====
+The ViaQ data model is now deprecated and is planned to be removed in future logging releases.
+====
```

Comment:
I'm afraid that's not right. IIRC ViaQ will only be deprecated once we GA OTEL

Reply:
+1   Additionally, a few lines above this is the statement "The ViaQ data model is the default and fully supported option..."

Reply:
Considering above suggestion for the Note, I believe it would be best to remove the Note for now to avoid confusion. Once OTEL reaches GA, then we can include a formal deprecation notice. Let me know if you have other thoughts. Thanks!